modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -0,0 +1,64 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distereoibution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_HPP
+#define OPENCV_GAPI_STEREO_HPP
+
+#include <opencv2/gapi/gmat.hpp>
+#include <opencv2/gapi/gscalar.hpp>
+#include <opencv2/gapi/gkernel.hpp>
+
+namespace cv {
+namespace gapi {
+
+enum class StereoOutputFormat {
+    DEPTH_FLOAT16,
+    DEPTH_FLOAT32,
+    DISPARITY_FIXED16_11_5,
+    DISPARITY_FIXED16_12_4
+};
+
+namespace calib3d {
```

Comment:
> calib3d

In OpenCV 5.0 calib3d is gone (see the "next" branch and PR #18940).
Used functionality is moved to "stereo" namespace.

Perhaps it makes sense to use new names for namespaces and functions instead.

Reply:
Thanks Alexander,

This namespace here is used only to hold an Operation's type -- the user code is supposed to use `cv::gapi::stereo` instead so we can move these types around easily with no visible impact

-D